spec/std/std.spec.js

```diff
@@ -0,0 +1,24 @@
+// import { StandardNotation } from '../../dist/ad-notations.esm.js';
+const { StandardNotation } = require('../../dist/ad-notations.umd.js');
+
+describe("Standard notation", function() {
+
+	const notation = new StandardNotation();
+
+	it(" less than 1000, 0 places", function() {
+		expect(notation.format(3, 0, 0)).toBe("3");
+		expect(notation.format(34, 0, 0)).toBe("34");
+		expect(notation.format(345, 0, 0)).toBe("345");
+	});
+	it(" more than 1e40, 0 places", function() {
+		expect(notation.format(3.45e40, 0)).toBe("35 DDc");
+		expect(notation.format(34.5e40, 0)).toBe("345 DDc");
+		expect(notation.format(345e40, 0)).toBe("3 TDc");
```

Comment:
Codacy found an issue: [The numeric literal '345e40' will have at different value at runtime.](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6241054)